### ISSUE TITLE
[feat] Support config binding on POJO class

### ISSUE THREAD

**Issue by syhily** [keykey7/confij#27]
The binding on interface is not serializable, supporting POJO would make it more acceptable.

**Comment by keykey7** [comment]
I agree with the pojo support. On the other side I can also totally picture serialization support for the interface proxies.
What's your actual usecase @syhily ?

**Comment by syhily** [comment]
POJO class is better for some streaming systems, I can also more logic through setter or getter. default value could be easily set as the same time.

I have finished the pojo class binding in my own forks. The pull request would be submitted a few days later.

**Comment by keykey7** [comment]
How fares your POJO-PR @syhily 😉 ? Still up for some code?
